library/backdrop

```diff
@@ -5,10 +5,10 @@ GitRepo: https://github.com/backdrop-ops/backdrop-docker.git
 
 Tags: 1.10.1, 1.10, 1, 1.10.1-apache, 1.10-apache, 1-apache, apache, latest
 Architectures: amd64, arm64v8
-GitCommit: 7cc189d199f92b981c1e2dc71c4d2d3aa4f4fa45
+GitCommit: d0e9a5029f1eb3eaf5bd06f74fd72403c069e31f
 Directory: 1/apache
 
 Tags: 1.10.1-fpm, 1.10-fpm, 1-fpm, fpm
```

Comment:
Just noticed these version numbers are *quite* outdated now (1.12 vs 1.10) -- you probably want to fix that, but in the interest of getting the security update out I'm going to merge this as-is (since these out of sync tag names aren't a new issue).